### ISSUE TITLE
One of the connected branches suddenly stopped deploying the changes uploaded to github

### ISSUE THREAD

**Issue by servappgit** [aws-amplify/amplify-console#1393]
** Please describe which feature you have a question about? **
The frontend enviroment in which we deploy one of branches wasn't updating the changes we uploaded to github.

** Provide additional details**
We got no errors, everything went smooth and we got green check boxes in the 4 stages (provision, build,deploy and verify). The github repository seems to be working fine, since we made sure that the changes were being uploaded correctly in the repository. The project works fine in local across all of our collaborators computers. Amplify seems to not be picking up the changes. We have several apps deployed in aws amplify and theyre working fine. This particular project is the only one having issues.

heres the branch name:master
Region:US-EAST-2

APP ARN:
arn:aws:amplify:us-east-2:305346604475:apps/d3le97war7ov3p

**Comment by garyleefight** [comment]
Can you be more specific? I saw builds are triggered for the app your shared in our logs.

**Comment by servappgit** [comment]
The web page doesnt change visually, eventho we've added new images and new funcionalities uploaded to our git repo, the web page remains in an older version. It's like it's not picking up the changes we've made.

**Comment by servappgit** [comment]
This is how it should look. https://drive.google.com/file/d/14pQ45xFXOFuuQtNQeHsxEHXWS7sjXoIz/view?usp=sharing

This is how it currently looks. 
https://drive.google.com/file/d/1_MIFbYV7jD-YH7FyycTDTxOP5kBce0sE/view?usp=sharing

So eventho we've added new changes, the web page remains in an older version. None os the changes we've uploaded to the connected repo from github to amplify are showing in amplify.
